staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics_test.go

```diff
@@ -226,6 +226,50 @@ func TestStorageSizeCollector(t *testing.T) {
 
 }
 
+func TestStorageSizeInUseCollector(t *testing.T) {
```

Comment:
This test case could be merged with `TestStorageSizeCollector` as a sub test.

Reply:
Ack, removed this test case and added the metric to the `TestStorageSizeCollector` test case